packages/wallet-ui/src/utils/utils.ts

```diff
@@ -218,3 +248,39 @@ export const shortenDomain = (domain: string, maxLength = 18) => {
   const shortenedPartLength = maxLength - ellipsis.length;
   return `${domain.substring(0, shortenedPartLength)}${ellipsis}`;
 };
+
+export function getTokenBalanceWithDetails(
+  tokenBalance: TokenBalance,
+  token: Erc20Token,
+  tokenUSDPrice: any,
```

Comment:
according to `addMissingPropertiesToToken` i think it is number | undefined

```suggestion
  tokenUSDPrice?: number,
```